src/RNGestureHandlerModule.web.ts

```diff
@@ -48,7 +49,7 @@ export default {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
       const GestureClass = HammerGestures[handlerName];
       // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-      HammerNodeManager.createGestureHandler(handlerTag, new GestureClass());
+      createGestureHandler.createGestureHandler(handlerTag, new GestureClass());
```

Comment:
While I agree that it would be better not to split `NodeManager`, this one only refers to `Hammer` implementation which we don't actively support right now. It may create a bit of a mess, but I believe it is worth it.

Have you tested old implementation @latekvo?

Reply:
I rolled back this splitting, as we fixed this particular circular dependency elsewhere.